src/DacpacTool/PackageAnalyzer.cs

```diff
@@ -34,18 +34,23 @@ public void Analyze(TSqlModel model, FileInfo outputFile, FileInfo[] analyzers)
                 var factory = new CodeAnalysisServiceFactory();
                 var settings = new CodeAnalysisServiceSettings();
 
-                if (analyzers.Length == 0)
-                {
-                    _console.WriteLine("DacpacTool warning SQLPROJ0001: No additional rules files found, consider adding more rules via PackageReference - see the readme here: https://github.com/rr-wfm/MSBuild.Sdk.SqlProj.");
-                }
-                else
+
+                if (analyzers.Length > 0)
                 {
-                    _console.WriteLine("Using additional analyzers: " + string.Join(", ", analyzers.Select(a => a.FullName)));
                     settings.AssemblyLookupPath = string.Join(';', analyzers.Select(a => a.DirectoryName));
                 }
 
                 var service = factory.CreateAnalysisService(model, settings);
 
+                var rules = service.GetRules();
+
+                if (!rules.Any(r => r.Namespace == "SqlServer.Rules" || r.Namespace == "Smells"))
+                {
+                    _console.WriteLine("DacpacTool warning SQLPROJ0001: No additional well-known rules files found, consider adding more rules via PackageReference - see the readme here: https://github.com/rr-wfm/MSBuild.Sdk.SqlProj.");
```

Comment:
Our readme is pretty extensive, it might be helpful to link directly to the relevant section: https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/blob/master/README.md#static-code-analysis

Reply:
@jeffrosenberg Great suggestion!